aptos-move/framework/src/natives/aggregator_natives/aggregator_v2.rs

```diff
@@ -55,18 +58,7 @@ pub const EINPUT_STRING_LENGTH_TOO_LARGE: u64 = 0x03_0008;
 /// and any calls will raise this error.
 pub const EAGGREGATOR_FUNCTION_NOT_YET_SUPPORTED: u64 = 0x03_0009;
 
-pub const STRING_SNAPSHOT_INPUT_MAX_LENGTH: usize = 256;
-
-/// Checks if the type argument `type_arg` is a string type.
-fn is_string_type(context: &SafeNativeContext, type_arg: &Type) -> SafeNativeResult<bool> {
-    let ty = context.deref().type_to_fully_annotated_layout(type_arg)?;
-    if let MoveTypeLayout::Struct(MoveStructLayout::WithTypes { type_, .. }) = ty {
-        return Ok(type_.name.as_str() == "String"
-            && type_.module.as_str() == "string"
-            && type_.address == AccountAddress::ONE);
-    }
-    Ok(false)
-}
+pub const DERIVED_STRING_INPUT_MAX_LENGTH: usize = 1024;
```

Comment:
Why we increase the size?

Reply:
for the unit test, to be able to detect gas charge changes... 

since we are charging by byte, and length is constant, I think it is fine to leave it bigger as well. 
Token has additional lower limit for the name itself, so this is only for custom ecosystem uses